code/index.html

```diff
@@ -1,22 +1,216 @@
+        <p class="read-more">[Read more] &rarr;</p>
+      </a>
+    </div>
+
+  </section>
+
+
+  <!-- SECTION 3 – the other news -->
+  <section id="section-3" class="section-3">
+
+    <!-- CARD - 1 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-01.jpg" alt="multi color paint">
+          <h3 class="text-on-pic">cool stuff</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Paint</span> - more is more.</p>
+    </div>
+
+    <!-- CARD - 2 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-49.jpg" alt="the desert">
+          <h3 class="text-on-pic">the desert</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Lost</span> in sandstorm.</p>
+    </div>
+
+    <!-- CARD - 3 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-33.jpg" alt="magenta neon lights">
+          <h3 class="text-on-pic">neon lights</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Awake</span> at nights.</p>
+    </div>
+
+    <!-- CARD - 4 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-23.jpg" alt="yellow tulips">
+          <h3 class="text-on-pic">tulips</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">From</span> a new perspective.</p>
+    </div>
+
+    <!-- CARD - 5 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-47.jpg" alt="japanese umbrella">
+          <h3 class="text-on-pic">osaka</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Greetings</span> from Japan.</p>
+    </div>
+
+    <!-- CARD - 6 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-27.jpg" alt="abstract painting">
+          <h3 class="text-on-pic">abstract</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Walls</span> in Dubai.</p>
+    </div>
+
+    <!-- CARD - 7 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-14.jpg" alt="brutalism architecture">
+          <h3 class="text-on-pic">brutalism</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Inspired</span> by architecture.</p>
+    </div>
+
+  </section>
+
+
+  <!-- SECTION 4 – footer -->
```

Comment:
You've done a great job with the footer - it's clean and fits well with the rest of the site!